### DISCUSSION TITLE
How to use Diffuser...?

### DISCUSSION THREAD

**Discussion by gomaxlou** [WASasquatch/was-node-suite-comfyui#90] [Q&A]
![image](https://github.com/WASasquatch/was-node-suite-comfyui/assets/37968309/a10ec32e-55d2-4f17-9b7f-75100bdb8dbf)

1. How to define the model_path ?
2. Can someone provide an example ? thanks

**Comment by WASasquatch** [comment]
The two nodes above expect a model already on your system in diffusers model format. They go into /models/diffusers path. 

However the Hub Model Down-Loader accepts a repo ID from huggingface.com. Next to repo names at the top of a model you'll see a copy icon. You can click that to easily copy the repo ID. Then you can paste it and download the model.

**Reply by gomaxlou** [reply]
1. Where do I go to download the diffuser model?
2.I don't understand, please provide screenshot

**Reply by WASasquatch** [reply]
![image](https://github.com/WASasquatch/was-node-suite-comfyui/assets/1151589/6b91aef7-5101-4402-8327-e4d41b07365c)
![image](https://github.com/WASasquatch/was-node-suite-comfyui/assets/1151589/e2d27db7-1906-4f2b-afdc-537145af881f)
![image](https://github.com/WASasquatch/was-node-suite-comfyui/assets/1151589/aafb9863-7dfd-4044-b05f-8630a3cd1f1c)

1. Copy repo ID from huggingface
2. Paste repo ID in `repo_id`
3. Run prompt to download and use the model.

Downloaded models will show up in Diffusers Model Loader

**Reply by gomaxlou** [reply]
i got error message :

Error occurred when executing Diffusers Hub Model Down-Loader:

Paths don't have the same drive

**Reply by WASasquatch** [reply]
> i got error message :
> 
> Error occurred when executing Diffusers Hub Model Down-Loader:
> 
> Paths don't have the same drive

I'd need to see the entire error, that is unfamiliar, but sounds like you're trying to use a different harddrive, which is apparently not supported.

**Reply by gomaxlou** [reply]
![image](https://github.com/WASasquatch/was-node-suite-comfyui/assets/37968309/33a55f18-ed23-4c33-944c-f2cd963cf1e4)

**Reply by WASasquatch** [reply]
Yeah, it says symlinks aren't supported for your system. Not sure where you are trying to save files, but try not using a symlinks.

**Reply by gomaxlou** [reply]
i try to fix it but fail , then manually download , ha,  finally it's ok , thanks
![diffUserHubVSBasic](https://github.com/WASasquatch/was-node-suite-comfyui/assets/37968309/f182bcea-9814-4c99-8dff-dfd03ae0ccf2)
